projects/addon-doc/components/main/main.component.ts

```diff
@@ -42,8 +62,29 @@ export class TuiDocMainComponent {
         @Inject(TUI_DOC_ICONS) private readonly icons: TuiDocIcons,
         @Inject(TuiThemeService) readonly theme: TuiThemeService,
         @Inject(TuiThemeNightService) readonly night: TuiThemeNightService,
+        @Inject(Router) private readonly router: Router,
     ) {}
 
+    ngAfterViewInit(): void {
+        this.router.events
+            .pipe(
+                delay(0),
+                filter((event): event is NavigationEnd => event instanceof NavigationEnd),
+                shareReplay({refCount: true, bufferSize: 1}),
+            )
+            .subscribe(() => {
```

Comment:
No need to subscribe just to push into a subject, it can be a piped Observable

Reply:
fixed